spec/controllers/links_controller_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
+RSpec.describe LinksController, type: :controller do
```

Comment:
FWIW: controller tests are going away in rails 5 (http://blog.bigbinary.com/2016/04/19/changes-to-test-controllers-in-rails-5.html explains the changes in detail - but not the rationale.  rspec is going to do the work that means you'll still be able to write them with rspec so when/if we upgrade we should be able to keep them, but the writing is on the wall that directly testing your controller is a no-longer-the-rails-way kinda thing to do so it might get harder and harder to do.

This is not to say that we shouldn't test our controllers, but that the specifics of how rails told you to test your controllers in the past (checking which template was rendered, checking which variables are assigned to share with the view, etc..) are not the way to do it.  Instead rails 5 suggests you test controllers with integration tests (feature specs in rspec parlance) where you don't reach inside to check the internals, but you can test the "full-stack" by checking what's on the rendered views.

So, all this to say, we have feature specs already so unless there's some complex branching route through the controller that we haven't covered in those, it's probably better to not introduce controller specs.